appshell/cef_window.h

```diff
@@ -249,3 +272,17 @@ class cef_window
 
     BOOL TrackNonClientMouseEvents(bool track = true);
 };
+
+
+#include <type_traits>
+#include <cstring>
+namespace from_cpp20 {
+	template<typename Dest, typename Source>
+		inline auto bit_cast(Source const & src) -> typename std::enable_if<sizeof(Source) == sizeof(Dest), Dest>::type
```

Comment:
there are a couple of more conditions to the enable_if in the std code https://en.cppreference.com/w/cpp/numeric/bit_cast see if it makes sense to incorporate that.

Reply:
I am not looking to implement a full proof back porting of C++20 `bit_cast`.
I can restrict it's uses further by asserting for pointer using `std::pointer_traits` or `std::is_pointer`.